t/151-exit-worker.t

```diff
@@ -0,0 +1,132 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use Test::Nginx::Socket::Lua;
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 2);
```

Comment:
Because the official test-nginx does not come with `shutdown_error_log` yet and your branch is currently passing the travis ci tests completely, which means this test plan is wrong since it does not count in the `shutdown_error_log` test in the first place.